package.json

```diff
@@ -134,5 +136,11 @@
     "webpack": "1.12.9",
     "webpack-cli": "1.0.0",
     "webpack-dev-server": "1.14.0"
+  },
+  "peerDependencies": {
+    "babel-polyfill": "6.9.0",
+    "babel-runtime": "6.9.0",
+    "react": "0.14.7",
+    "react-dom": "0.14.7"
```

Comment:
true, I'll use that... if you are using npm3, peerDependencies just issue warnings anyway... this will be useful for npm2 users